NavigationRail/src/androidTest/java/com/microsoft/device/display/samples/navigationrail/DetailTest.kt

```diff
@@ -212,14 +215,15 @@ class DetailTest {
     @Composable
     private fun DetailViewForFirstPlant() {
         NavigationRailAppTheme {
-            ItemDetailView(
+            TwoPaneNavScopeTest().ItemDetailView(
```

Comment:
I don't think we need `TwoPaneNavScopeTest`, since `ItemDetailView` is not under `TwoPaneNavScope`.

Reply:
ah good catch thank you, just updated the tests and confirmed they all pass now!